administrator/components/com_media/FocusStore/JSONFocusStoreClass.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_media
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Media\Administrator\FocusStore;
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\MVC\Controller\BaseController;
+use Joomla\Component\Media\Administrator\FocusStore\FocusStoreInterface;
+
+\JLoader::import('joomla.filesystem.file');
+
+/**
+ * Focus Store Class
+ *
+ * Used to set the focus point and save it into filesystem
+ *
+ * Used to get the focus point while rendering the page on the frontend
+ *
+ * @since  4.0.0
+ */
+class JSONFocusStoreClass implements FocusStoreInterface
```

Comment:
While we add "Interface" to interfaces and "Trait" to traits, we do *not* add "Class" to classes, as that's the default. So just "JSONFocusStore".